blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -156,7 +113,14 @@ public void testHttpAnnotationsIngestionHappyCase() throws Exception {
         final int batchSize = 1;
         final String tenant_id = "333333";
         String event = createTestEvent(batchSize);
-        postEvent( event, tenant_id );
+        HttpResponse response = postEvent( tenant_id, event );
+
+        try {
+            assertEquals( 200, response.getStatusLine().getStatusCode() );
```

Comment:
The try block is to make sure that the ```EntityUtils.consume``` is called.  If the response body is never read, the connection is never closed.  I'm using the try-finally blocks to ensure that it gets called.